common-arcana.lic

```diff
@@ -153,7 +155,7 @@ module DRCA
   end
 
   def get_runestone(runestone, settings)
-    return if (DRC.left_hand || DRC.right_hand) =~ /runestone/
+    return if DRC.left_hand =~ /runestone/ || DRC.right_hand =~ /runestone/
```

Comment:
💡 I think you can use the utility method here too for checking if runestone is in your hands
```suggestion
    return if DRCI.in_hands?('runestone')
```